src/System.Private.Xml/src/System/Xml/Xsl/Xslt/XPathPatternParser.cs

```diff
@@ -164,7 +164,7 @@ private QilNode ParseRelativePathPattern()
         {
             if (++_parseRelativePath > MaxParseRelativePathDepth)
             {
-                if (LocalAppContextSwitches.LimitXPathComplexity)
```

Comment:
What I meant was to have the class XmlAppContextSwitches renamed to LocalAppContextSwitches.

I know I suggested renaming this, but I think I would like to keep the same name as in Desktop.